Gruntfile.js

```diff
@@ -176,7 +176,8 @@ var _              = require('lodash'),
                     ui: 'bdd',
                     reporter: grunt.option('reporter') || 'spec',
                     timeout: '15000',
-                    save: grunt.option('reporter-output')
+                    save: grunt.option('reporter-output'),
+                    require: ['core/server/monkey-patch']
```

Comment:
yes a rename is a good shot, but i coudn't think of a name. any suggestions?
something like `overrides`?